product-histories.md

```diff
@@ -1,5 +1,7 @@
 # Product Histories
 
+For a concise summary of this section, see the tabular data of our products, when they were launched, and when they "retired" [here](https://github.com/basecamp/data_scratchpad/blob/master/miscellany/data/product_dates.csv).
```

Comment:
This handbook is public, so we try mostly not to link to other data sources that are private, when we can avoid it. I would either nix this or put a summary table in right here that's a copy of that.